benchmarks/recursive_computation/config.yaml

```
@@ -0,0 +1,46 @@
+shallow: &base
+  max_depth: 10
+  enable_sleep: false
+  sleep_duration: 0.1
+  profiler_enabled: false
+
+medium: 
+  <<: *base
+  max_depth: 50
+
+deep:
+  <<: *base
+  max_depth: 100
```

```suggestion
  max_depth: 300
```
keep this value in sync with the one you used in deep-profiled